src/utils/src/types/project-types.ts

```diff
@@ -77,3 +77,79 @@ export const createProjectPayloadSchema = {
 } as const;
 
 export type CreateProjectPayload = FromSchema<typeof createProjectPayloadSchema>;
+
+export const projectEditInputSchemaBody = {
```

Comment:
Is there any reason to not utilize the abstractions provided here? https://github.com/Northeastern-Electric-Racing/PM-Dashboard-v2/blob/main/src/utils/src/types/api-utils-types.ts

You might need to fast-forward your branch and fix merge conflicts to get the latest stuff from main. The abstractions should preserve all functionality and greatly simplify the code. It's being used in other payload schema definitions.